random_menu.rb

```diff
@@ -0,0 +1,65 @@
+# random_menu assignment from 02.08.2017
+# by Tamiko Terada
+
+puts "OH-SO-SEATTLE MENU GENERATOR"
+# create new array of 10 descriptors
+descriptor = [
+  "pan-seared",
+  "sous-vide",
+  "farm-raised",
+  "organic",
+  "local",
+  "rosemary-infused",
+  "a duet of",
+  "crispy",
+  "fire-roasted",
+  "ethically-raised"
+]
+# create new array with 10 meats
+meat = [
+  "duck breast",
+  "dungeness crab",
+  "venison",
+  "filet mignon",
+  "rabbit",
+  "sockye salmon",
+  "Wagyu beef sirloin steak",
+  "Carlton Farm pork",
+  "lamb shank",
+  "albacore tuna"
+]
+# create new array with 10 pairing items
+pairing = [
+  "hand-foraged wild onions",
+  "summer squash succotash",
+  "tri-colored kale",
+  "wild woodland mushrooms",
+  "oven-roasted beets",
+  "garlic and chive aioli",
+  "grilled radicchio",
+  "honey-roasted sunchokes",
+  "frangelico-blackberry compote",
+  "butternut squash rissoto"
+]
+# replace each array with shuffled array
+descriptor.shuffle!
+meat.shuffle!
+pairing.shuffle!
```

Comment:
Nice use of shuffle! This is a clever way to get a random order without destroying the array as you go along.